bower.json

```diff
@@ -33,7 +33,8 @@
   },
   "dependencies": {
     "angular": "1.2.25 - 1.3.*",
-    "patternfly": "1.1.1 - 1.1.4"
+    "patternfly": "1.3.0",
+    "c3": "https://github.com/masayuki0812/c3.git#0.4.11-rc1"
```

Comment:
I generally list the entire GIT repo path if the library isn't well known. This way it is clear to what is being pulled in